README.md

```diff
@@ -15,18 +15,29 @@ The tempest snap can be installed directly from the snap store:
 
     sudo snap install [install-OPTIONS] tempest
 
-By default, tempest snap will be installed from the stable channel of the latest OpenStack
-release, e.g. `2023.1/stable` if `antelope` is the most recent release, unless a different
-channel is explicitly specified. This channel is generally compatible with actively developed
-OpenStack releases thanks to the relatively flexible alignment between Tempest and its plugins
-with OpenStack versions. 
+## Compatibility matrix
 
-For a more tailored installation, users have the option to choose the channel that is designed
-for their cloud environment. For example, in a `yoga` OpenStack cloud, users can install tempest
-snap from the `yoga/stable` channel:
+With version numbers not being aligned between Tempest, the various Tempest
+plugins, and OpenStack releases, it is not immediately obvious which set of
+components are compatible with each other.
+
+The Tempest snap simplifies testing OpenStack clouds by providing
+release-specific channels, which package all the correct Tempest and Tempest
+plugin versions for the relative OpenStack release.
+
+The default channel for this snap is the stable channel for the latest current
+OpenStack release, e.g. `2023.1/stable` if Antelope is the most recent release.
+To test older clouds, please specify an alternate channel. For example, in a
```

Comment:
non-blocking: now that Bobcat is the latest release and we are already making changes to the README, maybe it would be better if we use that as the example?

Reply:
I did consider it, but decided against it due to Bobcat's short support lifecycle: https://ubuntu.com/about/release-cycle#openstack-release-cycle